tests/ci/docker_images/linux-arm/amazonlinux-2_accp_base/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM arm64v8/amazonlinux:2
+
+SHELL ["/bin/bash", "-c"]
+
+RUN set -ex && \
+    yum -y update && yum install -y \
+    cmake \
+    cmake3 \
```

Comment:
instead of doing the `cmake3`-autodetect workaround, would could we just install cmake at version 3? or perhaps use [update-alternatives](https://linux.die.net/man/8/update-alternatives) to manage which executable `cmake` command points to?

overall, this is just kind of a nitpick on my part -- i don't have any real issue with doing the autodetect in build.gradle, it just seems like a  slight conflation of concerns between "how to build the thing" (i.e. build.gradle) and "what to build the thing with" (i.e. our Dockerfiles).

Reply:
Unfortunately on AL2 & centOS, the version of `cmake` installed is always capped at `2.8.x`. The OS default only provides Cmake versions over 3 with the `cmake3` command. The `update-alternatives` method is true from a development environment standpoint. 
The main reason I thought of configuring `cmake3` auto-detection was thinking of customers using ACCP on these platforms. We used to directly call the `cmake` command within `build.gradle` and this would force all customers on these platforms to point to the correct `cmake/cmake3` executable themselves. Using this method would let them use ACCP out of the box without any extra configurations.

Reply:
> Unfortunately on AL2 & centOS, the version of cmake installed is always capped at 2.8.x. The OS default only provides Cmake versions over 3 with the cmake3 command

ah, bummer. that's an unfortunate restriction, but I'm sure there are reasons.

agreed that better usability is worth trading for any aesthetic nits I may have :)